src/index.html

```diff
@@ -15,8 +15,98 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="header-top-logo"
+        />
+      </a>
+      <nav class="navigation">
+        <ul>
+          <li>
+            <a
+              href="#apple"
+              class="nav-link is-active"
+            >
+              apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#samsung"
+              class="nav-link"
+            >
+              samsung
+            </a>
+          </li>
+          <li>
+            <a
+              href="#smartphones"
+              class="nav-link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li>
+            <a
+              data-qa="hover"
+              class="nav-link active"
+              href="#copmuters"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li>
+            <a
+              href="#gadgets"
+              class="nav-link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li>
+            <a
+              href="#tablts"
```

Comment:
There is a typographical error in the href attribute. 'tablts' should be corrected to 'tablets'.